styles.css

```diff
@@ -3,6 +3,7 @@
   font-family: 'Roboto', sans-serif;
   color: #fff;
   text-decoration: none;
+  margin: 0;
 }
 
 body {
```

Comment:
- [ ]  the Display should be flex and it should be applied on the section tag that supposed to be inside the body tag see the comment with the screenshot for more details